src/main/java/duke/task/TaskType.java

```diff
@@ -0,0 +1,7 @@
+package duke.task;
+
+public enum TaskType {
+    TODO,
+    DEADLINE,
+    EVENT;
```

Comment:
I think according to the coding standard, enum names should be written in PascalCase!

Reply:
I am not really sure on this myself but I thought enum name refers to TaskType instead. I will double check!